fedimint-rocksdb/src/lib.rs

```diff
@@ -31,10 +32,10 @@ impl RocksDb {
 }
 
 impl RocksDbReadOnly {
-    pub fn open_read_only(db_path: impl AsRef<Path>) -> Result<RocksDbReadOnly, rocksdb::Error> {
+    pub fn open_read_only(db_path: String) -> Result<RocksDbReadOnly, rocksdb::Error> {
```

Comment:
Why not `PathBuf`? I'm also not a fan of hiding this change in this commit, could use some explanation in a commit message.

Reply:
Let me see if I can fix this before merging

Reply:
Sorry I was being dumb, these changes aren't necessary. Reverted all changes in `fedimint-rocksdb`